style3.css

```diff
@@ -1,22 +1,21 @@
 /* Import  font from google fonts*/
-@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap');
-
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
-
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@600&display=swap');
```

Comment:
27/09/2021: I apply all the font that the music database is decided to use because I want to make it consistence